### DISCUSSION TITLE
LUTs for using WIDE with Log 400 (GP-Log), Log 113 (Protune Flat) and Native/Vibrant stock profiles.

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#627] [General]
The Labs feature WIDE is for a wider color gamut, awesome, until you want to match the look of stock GoPro footage.  These LUT help convert the GoPro wide gamut to Rec709 color.

LUTs - https://www.dropbox.com/scl/fi/jl4kuwcwekbb5exdsta6r/WIDE_Convertion_LUTs.zip?rlkey=zswcgy0aejn2f1sn80q2m2xuw&dl=1

WIDE_Flat_2_709_Flat.cube - Converts WIDE shot with Protune Flat to Flat with Rec709 color (preserves the curve.)    
WIDE_Flat_to_Rec709.cube - Converts WIDE shot with Protune Flat to Rec709 transfer function and color.  
WIDE_LOG400_to_709_LOG400.cube - Converts WIDE shots with GPLog/LOGB=400 to Rec709 color (preserves the curve.)  
WIDE_LOG400_to_Rec709.cube - Converts WIDE shot with GPLog/LOGB=400 to Rec709 transfer function and color.  
WIDE_Natural_to_Rec709.cube - Converts WIDE shot Natural/Vibrant to Rec709 color (preserves the curve.)

**Comment by lyricsdude** [comment]
thank you so much!!! amazing!!! :)

**Reply by lyricsdude** [reply]
oh wait it's not the creative look right?

**Reply by lyricsdude** [reply]
can you please create a LUT from WIDE to the creative look? :)

**Reply by dnewman-gpsw** [reply]
Pure technical, you add to creative flair.  These remaps WIDE=1 to Rec709 color from a range of different capture modes. 

P.S. No. I'm not on a creative LUT designer, these are engineered to be accurate, but from these it is much easier to make you own creative looks/LUTs. These is only one technical LUT per mode, but a billion create choices.

**Reply by lyricsdude** [reply]
but who wants to play with colors that don't match reality? what the sensor sees is different than what the eye sees. the gopro creative look is what my eyes see. DJI, sony, canon, you just adjust the gamma curve from the log/flat mode and the colors look good.

With WIDE=1 to rec709 blues and yellows are way too saturated, skin tones don't look right. with the creative look it looks great :)

**Reply by dnewman-gpsw** [reply]
Can you share your examples because with colored charts everything is coming up aligned correctly here.  Of course, using wide gamut is not for those who want GoPro color. It's for those who want to match other camera systems, or mix footage in commercial productions. If you want to be get the GoPro look, best way is choose the GoPro look in camera. That's the easiest way, also recommend by the GoPro media team. So what I'm missing is what workflow you're trying to achieve to have GoPro color like the camera, but you don't want to choose it in the camera.

**Reply by lyricsdude** [reply]
I like the creative look colors from the Gopro but I don't like the tone mapping basically. And TONE=3 is underexposed and applying exposure in post makes it look washed out.

**Reply by dnewman-gpsw** [reply]
I will look see if there is a tone mapping strength for LTM.  GTM is part of the look, without that colors will not be the same.  oMTONE=1 not work sufficiently?

**Reply by lyricsdude** [reply]
Thank you. TONE=1 is too harsh also. TONE=3 with normal exposure would be great. I try to record 2 samples. 1 with WIDE=1 and rec709 LUT and 1 with TONE=3 (what my eyes actually see).

**Reply by lyricsdude** [reply]
The colors are fairly accurate with the LUT, just some colors the saturation/luminance is too much.

**Reply by dnewman-gpsw** [reply]
Then turn down the hot colors, and export a new LUT, most editing tools can do that.

**Reply by lyricsdude** [reply]
but it's all over the place. yellow scissor and blue router too saturated. my bright orange shirt undersaturated with weird colors. My skin tones don't look normal. In TONE=3 it looks all very great :)

**Reply by lyricsdude** [reply]
is there no possibility a LUT WIDE shot with Protune Flat to Flat with "Creative Look" color? Or normal protune flat to "creative look"?

**Reply by dnewman-gpsw** [reply]
I expect they can be created, but I don't create those.  Ask https://www.leeminglutpro.com/

**Reply by RikshaDriver** [reply]
Is there any technical info on what type of tone mapping curve is applied?

**Reply by dnewman-gpsw** [reply]
No, and I'm sure that is internal IP.  GTM and LTM are scene dependent and dynamic, hard/impossible to emulate with just LITs.

**Reply by lyricsdude** [reply]
is WIDE exactly the same on Gopro 11 and 12? or there might be slight differences in some colors? On which one these LUTs work best?

**Reply by dnewman-gpsw** [reply]
It is exactly the same sensor.  If you select Flat with WIDE=1, they should be effectively identical.  The LUTs are each designed to match the in-camera options you select.

**Reply by lyricsdude** [reply]
thank you. I currently have the Gopro 11. Is there any reason to upgrade to the Gopro 12 when I use Gopro Labs?

**Comment by LeonardSander** [comment]
Hi, would you mind releasing LUTs that convert to Davinci Intermediate Gamma & Gamut please? Specifically i would need WIDE shot with GPLog/LOGB=400 to Davinci. I would like to heavily grade my footage in Davinci Resolve and i am confused how i should work with these LUTs otherwise. Use the Rec709 LUT at the end of my grade and use incorrect grading controls for given Gamma/Gamut before (since Davinci does not know either logb400 Gamma or Hero11 sensor native gamut) OR use the Rec709 LUT before the color grade which would probably be limiting for the end result.

**Reply by dnewman-gpsw** [reply]
I'm sure they exist already.  Third parties have created IDTs for GP-Log and Labs WIDE LOGB modes.

One example (which I have never used) https://xtremestuff.net/gp-tune-transform-manual/

**Reply by LeonardSander** [reply]
Well gp tune is the only option i found that supports wide/logb400 to davinci. And i don't necessarily want to pay for it😅

**Comment by Pax709** [comment]
I have a technical question, why in wide=1 the red color is not true i.e. too orange, no Lut changes this, while trying to restore the correct red color spoils the skin color, what is the explanation for this?, I get the impression that "wide" and skin color is a bumpy road.

**Reply by dnewman-gpsw** [reply]
"no Lut changes this"  - why not?  WIDE is simply not applying the color matrix in camera, you can apply the color matrix within a LUT, and this is what the latest GP-Log LUTs do.

**Reply by Pax709** [reply]
The problem is that these Lutes do not solve the problem of bad red color, why is that, have you ever done a comparison after converting wide to Rec709 vs Gorpo Natural or Vibrant?

I suggest you take a recording of something red in GOPRO NATURAL and in FLAT/LOGB400, WIDE=1, overlay GP-Log LUTs and compare the color, clearly it will differ.

**Reply by dnewman-gpsw** [reply]
GoPro Natural does other processes, like secondary color correction. The LUT aren't directly trying to match that, as those are dynamic functions like tone-mapping (scene dependent.) LUT are rather primitive in their operations and abilities. The point is all the same data is there, red and skin tones can be separated. How is more a colorist question, which I am not.   have used a MacBeth Chart and just let software do the calibration.

**Reply by Pax709** [reply]
I understand, but calibrating the colors should pretty much agree, they don't agree for some reason, probably the problem is running not on raw data but already processed H265. 

I used paid Lut and from GoPro, both suffer from the same thing, if only you could do your comparison and confirm that there is a bad red color.

**Reply by dnewman-gpsw** [reply]
Maybe I just don't see it, I shoot WIDE all the time and happily create the looks I need.  But generally I don't use LUTs, I start in a flat wide image, and pull it until it looks good.

**Reply by Pax709** [reply]
I "wasted" 😉a lot of time testing, and I have to admit that it is not as bad as I initially thought. I tested mainly FLAT/LOGB400/WIDE=1, I used two LUT boards, one from Dropbox ( from your link) and the other from GOPRO website, this GPLOG CREATIVE_2.0. 

Generally they are fine, only why one is dark and the other is bright, however, should I set the minimum - 1EV to LOG 400 and I set - 0.5 ? 

One of these LUTs, has a nice strong blue saturation, so that there may be problems with the blue color , but I will not confirm this in the field for now, while in my test the blue tones degraded, did you intentionally set the blue this way, or did the software do it this way? 

I also noticed that wide=1 creates a cleaner image and more "sharp", but this is probably due to the color presentation, as if more color contrast, less glow. I would appreciate it if you would elaborate on this and explain if I am wrong, because I can see the advantage of WIDE1 worth using.

**Reply by dnewman-gpsw** [reply]
WIDE=1 only turns off the 3x3 color saturation matrix, for a wider gamut.  The resulting image looks less noisy, in the desaturated state, as gaining up saturation also increases noise.  I don't know why it would look more sharp.  The luma signal is cleaner, maybe that is easier to run sharpening upon -- guessing.

**Reply by Pax709** [reply]
LEFT     -  NATURAL REC709
RIGHT  -   GPLOG_Creative_EV0 19.12.2023

![received_410130354980015.png](https://github.com/gopro/labs/assets/141012631/23a46a88-7771-491e-93d8-297c941a1eb5)

I apologize for the poor picture, now I do not have access to the original, but you can see the subtle differences in it. Could you refer to your LUTs from the previous question?

**Reply by dnewman-gpsw** [reply]
Natural and Log with LUT shouldn't look the same, and one uses tone-mapping.

**Reply by Pax709** [reply]
Natural Rec709 is the original file from the camera, the second file is from the superimposed Lut

**Reply by dnewman-gpsw** [reply]
Well that is confusing labelling.  I don't know what the point this demonstrates.

**Reply by Pax709** [reply]
This is a demonstration of the superiority of the wide 1 mode with the Natural file, not much can be done to match it on the right. Is the only difference between wide 1 and rec the colors other than gamut sRgb, or is there more to it than that?

**Reply by dnewman-gpsw** [reply]
No idea how this shows that.  WIDE=1 just turns off the color matrix, that is literally it.

**Reply by Pax709** [reply]
And what is the idea of using WIDE1 mode to profile Natural and then converting to Rec709, what do we gain?

**Reply by dnewman-gpsw** [reply]
Options. You can do it.  And you get to control how a color saturates.

**Reply by Pax709** [reply]
You're right, that's why with your Lut we get better colors after conversion to REC709, hence the better color contrast, I don't know if I'm calling it right.
